src/components/UI/Buttons/buttons.js

```diff
@@ -1,7 +1,10 @@
 import "./buttons.css";
 
 function NormalButton(props) { 
-    return <button className="normal-btn" style={{backgroundColor: `${props.color}` && "red"}} >{ props.name}</button>
+    return <button className={`normal-btn ${props.class}`}
+        style={{ backgroundColor: `${props.color}` || "red", padding: `${props.padding}` || "3px" }}
+        onClick={props.click}>
+        {props.name}{props.icon}</button>
```

Comment:
Suggestion: You can destructuee the individual properties from props object

Reply:
ok